test/e2e/storage/drivers/csi.go

```diff
@@ -246,14 +245,6 @@ func (h *hostpathCSIDriver) PrepareTest(f *framework.Framework) (*storageframewo
 		NodeName:                 node.Name,
 	}
 
-	// Disable volume lifecycle checks due to issue #103651 for the one
```

Comment:
so by removing this, the reconstruction part for csi-host-path driver will be tested ?

Reply:
Right, this check was added in https://github.com/kubernetes/kubernetes/commit/f1e1f3a416b70bafadf961518c330ce3b1b5459a and https://github.com/kubernetes/kubernetes/commit/e99b945b17823137a49176cb3561a756e9fd9a70 to work around https://github.com/kubernetes/kubernetes/issues/103651#issuecomment-887227562
So removing this check allows csi-hostpath to be tested again, and I can see those tests passing for csi-hostpath in [pull-kubernetes-e2e-gce-storage-disruptive](https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/108997/pull-kubernetes-e2e-gce-storage-disruptive/1532097553804300288/) now.